builder/files/etc/skel/.profile

```diff
@@ -1,5 +1,10 @@
 # ~/.profile: executed by Bourne-compatible login shells.
 
+if [ "$USER" == "pirate" ]; then
+  PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/games:/usr/games"
```

Comment:
Maybe it's better to do PATH=$PATH+"missing paths"? What if any of the defaults change?